bin/hdfcoinc/pycbc_coinc_statmap

```diff
@@ -211,13 +211,13 @@ if 'name' in all_trigs.attrs:
 # Incorporate hierarchical removal for any other loud triggers
 logging.info("Beginning hierarchical removal of foreground triggers.")
 
-# Step 1: Create arrays to use for reference when storing ifar and fap of
-#         foreground triggers that get hierarchically removed.
+# Step 1: Create a copy of foreground trigger ranking statistic for reference
+#         in the hierarchical removal while loop when updating ifar and fap of
+#         hierarchically removed foreground triggers.
```

Comment:
https://github.com/ligo-cbc/pycbc/blob/master/bin/hdfcoinc/pycbc_coinc_statmap#L214-L216

Are identical here, so I have no idea why Github thinks this is a new edit?